e2-studio-star-rx72n-firmware/libs/rx_usb_comm/src/rx_usb_comm.c

```
@@ -1252,18 +1165,76 @@ rx_usb_comm_receive(rx_usb_comm_handle_t* handle, rx_frame_t* frame, const uint3
+    }
+
+    /* PONG / RESET_ACK: consume silently, loop for next frame */
+    if (frame->header.type == k_frame_type_pong || frame->header.type == k_frame_type_reset_ack) {
+      continue;
+    }
+
+    /* Data frame (COMMAND, RESPONSE): validate sequence via session, return to caller */
+    rx_session_validate_result_t validate_result = k_session_validate_fail;
+    rx_err_t                     validate_err =
+      rx_session_validate_rx(handle->session, frame->header.sequence, &validate_result);
+    if (validate_result == k_session_validate_fail) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)